src/components/molecules/search-results-list/search-results-list.jsx

```diff
@@ -93,14 +96,16 @@ const SearchResultsList = ({
 					{i18n.results[language]} {fromPage}-{toPage} {i18n.of[language]}{' '}
 					{results.totalResults} {i18n.for[language]}: {keyword}
 				</h4>
-				<SearchResultsPager
-					testid={testIds.RESULTS_PAGER_TOP}
-					current={currentPage}
-					totalResults={results.totalResults}
-					resultsPerPage={resultsPerPage}
-					language={language}
-					keyword={keyword}
-				/>
+				{!hidePager && (
```

Comment:
can we reverse the logic on this to make it a positive case?  like `showPager &&`